CONTRIBUTING.md

```diff
@@ -77,50 +77,52 @@ Then run the whole integ test suite against Cadence+Temporal service by this com
 
 Or run `make temporalIntegTests` if you don't want to wait :/
 
-### Option 2: Run with your own Temporal service
+To run the server, if you are in an IDE, you can run the main function in `./cmd/main.go` with argument `start`.
 
-NOTE: For local testing it's recommended to use [temporal docker-compose](https://github.com/temporalio/docker-compose)
+Alternatively, you could just build the binary by `make bins` and then run `./iwf-server start`
```

Comment:
Or, directly using the command line in a terminal under the iwf folder:
```
 go run cmd/server/main.go start
```

Reply:
good idea. I never used this but just tried it and it works :D